lib/oban.ex

```diff
@@ -400,11 +400,20 @@ defmodule Oban do
 
   ## Node Name
 
-  When the `node` value hasn't been configured it is generated based on the environment:
+  When the `node` value has not been configured it is generated based on the environment:
+
+  1. If the local node is alive (e.g. in a distributed system, or when running from a mix release)
+     the node name is used
+  2. In a Heroku environment the system environment's `DYNO` value is used
+  3. Otherwise, the system hostname is used
+
+  Note: When running a mix release on a Heroku node, the node is alive even if not part of a
+  distributed system. In order to use the `DYNO` value, configure the node value using runtime
+  configuration via `config/runtime.exs:
+
+        config :my_app, Oban,
+          node: System.get_env("DYNO") || :inet.gethostname() |> elem(1) |> List.to_string()
```

Comment:
The fallback portion after the `||` isn't explained in the prelude, and I don't think it's necessary to explain how to set the `node`.

```suggestion
        config :my_app, Oban,
          node: System.get_env("DYNO", "nonode@nohost")
```